buffer/variables.tf

```diff
@@ -125,3 +126,15 @@ variable "billing_account_ids" {
   description = "List of Google billing account ids to allow Resource Buffer Service to use"
   default     = []
 }
+
+variable "parent_folder_id" {
+  type        = string
+  description = "Parent folder under which to create all pool-specific folders. If empty, no folders will be created."
+  default     = ""
+}
+
+variable "pool_list" {
```

Comment:
nit: maybe `pool_names` or `pool_ids` or just `pools`? Don't know that `_list` is particularly informative and we don't suffix that for other lists.

Reply:
Went with pool_ids to match the config in RBS.